app/api/datastore/entities/v1/crime_application.rb

```diff
@@ -2,10 +2,76 @@ module Datastore
   module Entities
     module V1
       class CrimeApplication < Grape::Entity
-        expose :application, merge: true
```

Comment:
Why not do this anymore?
It feels more clunky having to expose one by one everything?